ui/src/components/Dialog.tsx

```diff
@@ -9,7 +9,9 @@ export default function Dialog({
 }: DialogPrimitive.DialogProps) {
   return (
     <DialogPrimitive.Root {...props}>
-      <DialogPrimitive.Overlay className="fixed inset-0 z-40 transform-gpu bg-black opacity-30" />
+      <DialogPrimitive.Portal>
```

Comment:
I think you might want this to surround children too like they do in the example here: https://www.radix-ui.com/docs/primitives/components/dialog

Reply:
Pushing the fix for this now.